codex-rs/cli/Cargo.toml

```diff
@@ -36,3 +36,11 @@ tokio = { version = "1", features = [
 ] }
 tracing = "0.1.41"
 tracing-subscriber = "0.3.19"
+predicates = "3.1.3"
```

Comment:
Is this needed? If so, alpha sort and make sure it shouldn't go in `dev-dependencies`?